docs/design/server42/src/server42_mk3/include/server42/statistics.proto

```diff
@@ -1,13 +1,14 @@
 syntax = "proto3";
 
-package org.openkilda;
+package org.openkilda.server42.stats.messaging.flowrtt;
 
 
 message FlowLatencyPacket {
-    bytes flow_id = 1;
-    int64 t0 = 2;
-    int64 t1 = 3;
-    uint64 packet_id = 4;
+    string flow_id = 1;
+    bool direction = 2;
+    int64 t0 = 3;
+    int64 t1 = 4;
```

Comment:
I don't like names t0 and t1. Maybe use some meaningful names?

Reply:
We already use that notation in project, so it's not a good place to rework naming.